_posts/2015-03-26-webcomponents-0.6.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "webcomponents.js 0.6.0 release"
+date:   2015-03-26 15:32:00
+categories: updates
+author: polymer-team
+excerpt: The 0.6.0 version of the webcomponent.js polyfills have been released - the first major release since the polyfills moved to their new home at [webcomponents.org](https://www.webcomponents.org).
+
+---
+
+Exciting news - the 0.6.0 version of the `webcomponents.js` polyfills has been [released](https://github.com/webcomponents/webcomponentsjs/tree/v0.6.0). A lot of critical fixes are in this release - highlights are described below.
+
+This also marks about 6 months since the polyfills were moved from the Polymer organization over to WebComponents.org. Here are some stats comparing the Polymer/platform repository (the precursor to webcomponents.js) activity in the 6 months prior to the move, to the WebComponents/webcomponentsjs repository in the six months since its creation:
```

Comment:
As mentioned in mail, to reiterate the move, I think the cross-post to wc.org should be where the canonical release notes live and the version we land on the Polymer blog is the FYI version. That way folks know it's not (still) just a Polymer sub-project. We've had tons of community help etc.